File: Code/Disney/disney.xBandController/src/linux/IDMSLib/src/main/java/com/disney/xband/idms/lib/model/Guest.java
package com.disney.xband.idms.lib.model;

import java.util.List;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class Guest
{
	
	private long guestId;
	private String dateOfBirth;
	private String status;
	private String countryCode;
	private String swid;
	private String languageCode;
	private String emailAddress;
	private String parentEmail;
	private GuestName name;
	private String gender;
	private String userName;
	private String avatar;
	private int visitCount;
	private String IDMSTypeName;
	private List<Link> links;
	private long partyId;
	
	
	public Guest()
	{
		
	}
	
	@XmlElement(name="partyId")
	public long getPartyId()
	{
		return this.partyId;
	}
	
	@XmlElement(name="partyId")
	public void setPartyId(long partyId)
	{
		this.partyId = partyId;
	}
	
	
	@XmlElement(name="guestId")
	public long getGuestId() {
		return guestId;
	}
	
	@XmlElement(name="guestId")
	public void setGuestId(long guestId) {
		this.guestId = guestId;
	}
	
	@XmlElement(name="dateOfBirth")
	public String getDateOfBirth() {
		return dateOfBirth;
	}
	
	@XmlElement(name="dateOfBirth")
	public void setDateOfBirth(String dateOfBirth) {
		this.dateOfBirth = dateOfBirth;
	}
	
	@XmlElement(name="status")
	public String getStatus() {
		return status;
	}
	
	@XmlElement(name="status")
	public void setStatus(String status) {
		this.status = status;
	}
	
	@XmlElement(name="countryCode")
	public String getCountryCode() {
		return countryCode;
	}
	
	@XmlElement(name="countryCode")
	public void setCountryCode(String countryCode) {
		this.countryCode = countryCode;
	}
	
	@XmlElement(name="swid")
	public String getSwid() {
		return swid;
	}
	
	@XmlElement(name="swid")
	public void setSwid(String swid) {
		this.swid = swid;
	}
	
	@XmlElement(name="languageCode")
	public String getLanguageCode() {
		return languageCode;
	}
	
	@XmlElement(name="languageCode")
	public void setLanguageCode(String languageCode) {
		this.languageCode = languageCode;
	}
	
	@XmlElement(name="emailAddress")
	public String getEmailAddress() {
		return emailAddress;
	}
	
	@XmlElement(name="emailAddress")
	public void setEmailAddress(String emailAddress) {
		this.emailAddress = emailAddress;
	}
	
	@XmlElement(name="parentEmail")
	public String getParentEmail() {
		return parentEmail;
	}
	
	@XmlElement(name="parentEmail")
	public void setParentEmail(String parentEmail) {
		this.parentEmail = parentEmail;
	}
	
	@XmlElement(name="name")
	public GuestName getName() {
		return name;
	}
	
	@XmlElement(name="name")
	public void setName(GuestName name) {
		this.name = name;
	}
	
	@XmlElement(name="gender")
	public String getGender() {
		return gender;
	}
	
	@XmlElement(name="gender")
	public void setGender(String gender) {
		this.gender = gender;
	}
	
	@XmlElement(name="userName")
	public String getUserName() {
		return userName;
	}
	
	@XmlElement(name="userName")
	public void setUserName(String userName) {
		this.userName = userName;
	}
	
	@XmlElement(name="avatar")
	public String getAvatar() {
		return avatar;
	}
	
	@XmlElement(name="avatar")
	public void setAvatar(String avatar) {
		this.avatar = avatar;
	}
	
	@XmlElement(name="visitCount")
	public int getVisitCount() {
		return visitCount;
	}
	
	@XmlElement(name="visitCount")
	public void setVisitCount(int visitCount) {
		this.visitCount = visitCount;
	}
	
	@XmlElement(name="type")
	public String getIDMSTypeName() {
		return IDMSTypeName;
	}
	
	@XmlElement(name="type")
	public void setIDMSTypeName(String iDMSTypeName) {
		IDMSTypeName = iDMSTypeName;
	}
	
	
	@XmlElement(name="links")
	public List<Link> getLinks()
	{
		return this.links;
	}
	
	@XmlElement(name="links")
	public void setLinks(List<Link> links)
	{
		this.links = links;
	}
	
}

File: Code/Disney/disney.xBandController/src/windows/Test/Disney.xBand.IDMS/Guests/Guest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;
using Disney.xBand.IDMS.Links;

namespace Disney.xBand.IDMS.Guests
{
    [DataContract]
    public class Guest
    {
        [DataMember(Name = "guestId")]
        public long GuestID { get; set; }

        [DataMember(Name = "dateOfBirth")]
        public String DateOfBirth { get; set; }

        [DataMember(Name = "status")]
        public string Status { get; set; }

        [DataMember(Name = "countryCode")]
        public string CountryCode { get; set; }

        [DataMember(Name = "swid")]
        public string SWID { get; set; }

        [DataMember(Name = "languageCode")]
        public string LanguageCode { get; set; }

        [DataMember(Name = "emailAddress")]
        public string EmailAddress { get; set; }

        [DataMember(Name = "parentEmail")]
        public string ParentEmail { get; set; }

        [DataMember(Name = "name")]
        public GuestName Name { get; set; }

        [DataMember(Name = "gender")]
        public string Gender { get; set; }

        [DataMember(Name = "userName")]
        public string UserName { get; set; }

        [DataMember(Name = "avatar")]
        public string Avatar { get; set; }

        [DataMember(Name = "visitCount")]
        public int VisitCount { get; set; }

        //[DataMember(Name = "type")]
        //public string IDMSTypeName { get; set; }

        [DataMember(Name = "partyId")]
        public long partyID { get; set; }

        public Guest()
        {
        }
    }
}
